interfaces/builtin/browser_support.go

```diff
@@ -51,6 +51,9 @@ owner /{dev,run}/shm/{,.}com.google.Chrome.* rw,
 deny dbus (send)
     bus=session
     interface="org.gnome.GConf.Server",
+
+# Webkit uses a particular SHM names # LP: 1578217
+owner /{dev,run}/shm/WK2SharedMemory.* rw,
```

Comment:
If this rule must go in, we should add it to the browserSupportConnectedPlugAppArmorWithSandbox policy since that is only applied with 'allow-sandbox: true' and that is mediated by the base declaration (ie, it is reserved for trusted apps). I don't like this workaround though-- there will be consumers of qtwebkit, just like chromium content api, that we won't want to allow 'allow-sandbox: true' but that will need it for this one access.